lib/ex_auction.ex

```diff
@@ -51,30 +51,34 @@ defmodule ExAuction do
           | {:error, ExAuction.Auction.Bid.Error.t()}
           | {:error, :auction_not_found}
           | {:error, :argument_error}
-  def place_bid(%Auction{} = auction, %Auction.Bid{user_id: user, value: value} = bid)
-      when is_binary(user) and is_integer(value) do
+  def place_bid(%Auction{} = auction, %Auction.Bid{value: value} = bid)
```

Comment:
Why are we getting rid of user_id ?

Reply:
I have removed the `user_id` guard clause because it is almost impossible to know which data-type it should be.

I originally had an issue with it, because we are using `integer` for our user_id, and I thought in reality, other people could also use string (e.g. UUID) for their user_id, hence it's not realistic to choose one or either (integer or binary).